src/modalities/dataloader/create_packed_data.py

```diff
@@ -62,11 +63,11 @@ def __init__(
         self.tokenizer = tokenizer
         self.eod_token = eod_token
         self._token_size_in_bytes = self._get_required_num_of_bytes_to_repr(self.tokenizer.vocab_size)
-        encoded_eod_token = self.tokenizer.get_token_id(self.eod_token)
-        self._encoded_eos_token_as_bytes = self._encoded_token_to_bytes(encoded_eod_token)
+        eod_token_id = self.tokenizer.get_token_id(self.eod_token)
+        self._encoded_eos_token_as_bytes = self._encoded_token_to_bytes(eod_token_id)
```

Comment:
```suggestion
        self._encoded_eod_token_as_bytes = self._encoded_token_to_bytes(eod_token_id)
```
Use `eod` consistently instead of both `eos` and `eod`